packages/aws-cdk/README.md

```diff
@@ -559,6 +560,87 @@ This feature is currently in preview. Be aware of the following limitations:
   in the right order. The CLI will not help you import dependent resources in the right
   order, the CloudFormation deployment will fail with unresolved references.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+We make no guarantees about the outcome or stability of the Command nor it's output.
+
+Generates a CDK application from an existing CloudFormation template in any of the CDK supported languages. 
```

Comment:
This is the first sentence of that opening paragraph. Comparing this to `cdk deploy`'s entry in this file, notice how `cdk deploy` has a similar opening sentence, but then goes into a high level overview of the command before it gets into use cases.

Notice also how `cdk deploy` is organized into use cases, not flag entries. Notice also how the use cases are written in the format of:
 - problem statement
 - solution
 - example
 - details

A good example of this format is the [deployment progress](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk#deployment-progress) entry. It starts with:

> By default, stack deployment events are displayed as a progress bar with the events for the resource currently being deployed.

This tells user that all the stack deployment history is obscured by default. This is a *problem* because users may want the full stack history (we could improve this slightly by explicitly stating that the deployment history is obscured). The doc then shows a *solution*:

>Set the --progress flag to request the complete history which includes all CloudFormation events

and immediately follows with an *example* that shows how to use that solution:

```
$ cdk deploy --progress events
```

It then goes into *details* about flag usage.